code/index.html

```diff
@@ -1,30 +1,248 @@
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
+<html id="top">
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css">
+  <title>Nano Colors</title>
+  <!-- dont forget to add a css file and link it here! -->
+</head>
+
+<body>
+  <!-- Hero header with a bunch of animated pictures -->
+  <header class="header">
+    <div class="header__buzz">
+      <h2 class="header__buzz__word">be brave</h2>
+      <h2 class="header__buzz__word">be bold</h2>
+    </div>
+    <picture class="header__img header__img--30-0 header__img--animation-up header__img--animation-short">
+      <img class="header__img" src="./images/hearthair1920px.jpg" alt="woman with long flowing on the floor">
+    </picture>
+    <picture class="header__img header__img--20-50 header__img--animation-down header__img--animation-long">
+      <source media="(max-width: 600px) and (max-device-width: 600px)" srcset="./images/leftMobile.png ">
+      <img class="header__img" src="./images/leftDesktop.png" alt="Image description">
+    </picture>
+    <picture class="header__img header__img--20-50 header__img--animation-up header__img--animation-long">
+      <source media="(max-width: 600px)and (max-device-width: 600px)" srcset="./images/rightMobile.png">
+      <img class="header__img" src="./images/rightDesktop.png" alt="Image description">
+    </picture>
+    <picture class="header__img header__img--30-0 header__img--animation-down header__img--animation-short">
+      <img class="header__img" src="./images/flowyhair1920px.jpg" alt="pink haired woman tosses head back">
+    </picture>
+    <div class="header__logo">
+      <h1 class="header__logo__text"></h1>
+    </div>
+  </header>
+
+  <nav class="nav">
+    <!-- Sticky navbar under hero-->
+    <div class="nav-wrapper">
+      <div class="nav-wrapper__button"> <img src="./images/menu.png"></div>
+      <div class="nav-wrapper__menu">
+        <a class="nav-wrapper__menu__link" href="releases">New releases</a>
+        <a class="nav-wrapper__menu__link" href="products">Products</a>
+        <a class="nav-wrapper__menu__link" href="store">Online store</a>
+        <a class="nav-wrapper__menu__link" href="chart">Shopping chart</a>
+        <a class="nav-wrapper__menu__link" href="login">Login</a>
+      </div>
+    </div>
+  </nav>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+  <main class="main">
+    <!-- SIgnup form -->
+    <section class="signup">
+      <div class="signup-inner">
+        <h2>Sign Up!</h2>
+        <form class="signup-inner__form" action="https://httpbin.org/anything" method="POST">
+          <fieldset class="signup-inner__form__fieldset" class="sigup__fieldset" class="sipgnup__field">
```

Comment:
Hmm, if you need to add multiple amount of classes, you should just divide them by whitespace, not create new `class` attribute :

```
<fieldset class="signup-inner__form__fieldset sigup__fieldset sipgnup__field">
```

Needs to be fixed